contracts/gateway/OSTPrime.sol

```diff
@@ -49,20 +49,20 @@ contract OSTPrime is UtilityToken, OSTPrimeConfig, Mutex {
 
     using SafeMath for uint256;
 
-    /** Emitted whenever OST Prime token is converted to OST Prime base token. */
-    event TokenUnwrapped(
+    /** Emitted whenever OST is converted to coin. */
+    event OSTUnwrapped(
         address indexed _account,
         uint256 _amount
     );
 
-    /** Emitted whenever OST Prime base token is converted to OST Prime token. */
-    event TokenWrapped(
+    /** Emitted whenever coin is converted to OST. */
+    event OSTWrapped(
```

Comment:
Here it is slightly more debatable, but `TokenWrapped` is fine, and avoids an interface change.  The comment can be more explicit: "Emitted whenever the base coin OST is converted to EIP20 OST"